src/lib/holocene/textarea.svelte

```diff
@@ -120,8 +120,4 @@
   .count > .error {
     @apply text-danger;
   }
-
-  textarea {
-    @apply surface-primary dark:bg-transparent;
```

Comment:
Do we not want primary bg on textarea?

Reply:
It already has it, this style declaration is unnecessary because it's using [inline styles](https://github.com/temporalio/ui/pull/2599/files#diff-3c0342b14c6a3e4e46754c7373b2ed47b623592839e8e8eee008c4f5c70cbdc1R56-R59) and twMerge